pkg/util/oc/oc.go

```diff
@@ -82,6 +82,24 @@ func CreateNamespace(t test.TestHelper, ns string) {
 	shell.Executef(t, "oc new-project %s", ns)
 }
 
+func CreateNamespaces(t test.TestHelper, yaml string, data interface{}) {
+	t.T().Helper()
+	t.Logf("Creating namespaces")
+	ApplyTemplate(t, "default", yaml, data)
+}
+
+func DeleteNamespaces(t test.TestHelper, yaml string, data interface{}) {
+	t.T().Helper()
+	t.Logf("Deleting multiple namespaces")
+	DeleteFromTemplate(t, "default", yaml, data)
+}
+
+func UpdateSMMRMultipleNamespaces(t test.TestHelper, ns string, yaml string, data interface{}) {
+	t.T().Helper()
+	t.Logf("Creating smmr")
+	ApplyTemplate(t, ns, yaml, data)
+}
```

Comment:
This doesn't belong here. It's way too specific and not directly related to `oc`. I think this function should be in the test file where it's used. 

If we later see that we need it in other places, we can decide what to do later.